controllers/notification/.dockerignore

```diff
@@ -0,0 +1,4 @@
+# More info: https://docs.docker.com/engine/reference/builder/#dockerignore-file
+# Ignore build and test binaries.
+bin/
```

Comment:
dockerignore should not include `bin/`, which will cause docker build to fail to get the bin directory.